examples/react-todos/.meteor/packages

```diff
@@ -13,5 +14,6 @@ reactive-var
 meteorhacks:npm
 cosmos:browserify
 
+static-html
```

Comment:
another thing would be to create a client only package that we could serve from a static website and not from Meteor, but I guess that is out of scope (although might be important for React devs as well)